src/components/ReportCard.js

```diff
@@ -70,69 +94,59 @@ const StyledPill = styled.div`
   display: flex;
   justify-content: center;
   align-items: center;
-  width: 69px;
-  height: 28px;
+  width: 70px;
+  height: 30px;
   border-radius: 20px;
-  position: absolute;
-  left: 25px;
-  top: 25px;
-
-  ${minBreakpointQuery.medium`
-    left: 25px;
-    top: 25px;
-  `}
-
-  ${minBreakpointQuery.large`
-    left: 40px;
-    top: 40px;
-  `}
 `;
 
 const StyledImage = styled.img`
+  display: grid;
+  grid-area: 1 / 1 / 1 / 1;
+  position: relative;
+  border-radius: 15px;
   max-height: 100%;
   max-width: 100%;
+  object-fit: contain;
 `;
 
-const StyledHeader = styled.h3`
-  font-family: ${brandFonts.serif};
-  font-size: 25px;
-  font-weight: 700;
-  color: ${brandColours.tertiary};
-  line-height: 35px;
-  margin-bottom: 20px;
+const StyledContentOuter = styled.div`
+  position: relative;
+  display: grid;
```

Comment:
Use flex instead of grid here, this is why the text sits at the top instead of the bottom on;

![Screenshot 2024-02-14 at 14 49 14](https://github.com/tomkeyes-attachdigital/DiversityTravelHomePage/assets/60260005/54c29b77-7ad0-4b5d-8d75-f87b4fdde4ce)